bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/scheduler/ExpressionThreadPoolManager.java

```diff
@@ -38,8 +37,6 @@
 
     private final static Logger logger = LoggerFactory.getLogger(ExpressionThreadPoolManager.class);
```

Comment:
I know its not part of the PR but maybe you can change the logger to `privat final` right away.